runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -638,6 +651,35 @@ private boolean checkAnimationIdle(Duration waitDuration, Player local)
 		return false;
 	}
 
+	private boolean checkMovementIdle(Duration waitDuration, Player local)
+	{
+		if (lastPosition == null)
+		{
+			lastPosition = lastNotifiedPosition = local.getWorldLocation();
+			return false;
+		}
+
+		WorldPoint position = local.getWorldLocation();
+
+		if (lastPosition.distanceTo(position) == 0)
+		{
+			if (local.getAnimation() == IDLE
+				&& lastNotifiedPosition.distanceTo(position) != 0
```

Comment:
Can't `lastNotifiedPosition` just be replaced with a `boolean` like the other notify methods?

Reply:
It could, yeah; I've changed this now, thanks.